src/main/java/com/example/Main.java

```diff
@@ -1,7 +1,21 @@
 package com.example;
 
+/**
+ * Main class that contains void main
```

Comment:
Finish sentences with periods. Again, I can see what you write in the code. What does this class do? Why are we having it?